### ISSUE TITLE
Disable/Enable tab separation

### ISSUE THREAD

**Issue by c-sanchez** [lily-debug/extab#9]
An option to enable/disable the possibility to separate window tabs.

**Comment by lily-debug** [comment]
Is there a bug when separate window tabs?   why do you want this option?

**Comment by c-sanchez** [comment]
Oh, I had forgotten it but indeed, there is a bug by this, though I hope it can be fixed by keeping the option available.
I leave you a video on youtube replicating the bugs and a list with the minute when they occur.
https://youtu.be/-YZ-pqA099g

00:18 bug, problem with windows separation, this bug triggers the following list of bugs.
00:41 bug, the whole top bar of the windows disappears (title and standard window buttons)
00:48 bug, it is impossible to close the window normally
00:54 extab has crashed by close bugged window (notice how the tray icon disappears, sec 56)

I also leave you the dump that extab creates after the crash
[1.0.0.22-20210426-111231.zip](https://github.com/lily-debug/extab/files/6378423/1.0.0.22-20210426-111231.zip)

**Comment by lily-debug** [comment]
Yes, there is a bug when you drag the tab out of the window., I will fix it.
